models/src/main/java/com/vimeo/networking2/LiveQuota.kt

```diff
@@ -13,18 +13,25 @@ import com.vimeo.networking2.annotations.Internal
 @JsonClass(generateAdapter = true)
 data class LiveQuota(
 
-    /**
-     * Live streams quota data.
-     */
-    @Internal
-    @Json(name = "streams")
-    val streams: LiveStreamsQuota? = null,
+        /**
```

Comment:
Do you have special formatting on? When I format this file it does not add extra indentation (and I don't think Mohit's formatter adds the indentation either since he wrote these files), I think your formatter might be incorrectly configured for this project so the indentation should be reverted.